### DISCUSSION TITLE
Keyboard shortcut

### DISCUSSION THREAD

**Discussion by schamberg22** [Stellarium/stellarium#2784] [General]
Hi,
I write scrits, and i want to share some. 
I use "pauseScript" and i have define an alternative shortcut on my computer to resume execution script because the initial shortcuts don't work.
For other users, I want to know if it's possible to declare an alternative shortcut in the script? And how?

Thanks.

**Comment by alex-w** [comment]
F7?

**Reply by schamberg22** [reply]
I know to edit and modify keyboard shortcut.
I search a command to assign a shortcut to resume execution script after pauseScript in the writing of script.
The goal : other users don't need to edit an modifiy keyboard shortcuts before to launch a script.
Maybe it's not possible??

**Reply by alex-w** [reply]
Assignment 2 different shortcuts within one script is not possible. I don’t know how your script is working, but maybe you can implement trigger by state and use one shortcut?

**Reply by schamberg22** [reply]
Thanks Alex for yours replies.
For my scripts i  just use commands on this page : https://stellarium.org/doc/1.x/scripting.html.
I'm not a programmer. I don't know how to implement trigger in a script.
If you have an example to guide me.... i take.

**Comment by gzotti** [comment]
Note that pauseScript is no longer available in V1.* due to changes in the underlying Qt framework. You currently have to split your script into smaller ones. Sorry for the inconvenience.

**Reply by schamberg22** [reply]
I read there are some problems with the new version. I don't use it. My computer is a little older.
